app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/adapters/recycleradapters/RoomsAdapter.kt

```diff
@@ -27,19 +26,30 @@ class RoomsAdapter(private val availableRoomsList: ArrayList<DeviceConnectFragme
     }
 
     override fun onBindViewHolder(holder: ViewHolder, p1: Int) {
+        if (p1 == selectedIndex) {
+            holder.itemView.image_tick.background = context?.let { ContextCompat.getDrawable(it, R.drawable.ic_home_blue_24dp) }
```

Comment:
Which context you are using? It can be a non-null type. right?

Reply:
Just changed the type. It can be a non-null type now.